_tutorials/new-component/create-a-new-component-part-3.md

```diff
@@ -0,0 +1,532 @@
+
+### Supported Brands
+
+Origami components may support one or more brand. The brands a component supports are defined along with other component details in [`origami.json`](/spec/v1/manifest/#brands), by the `brands` property. If `brands` is not set the component is "unbranded" and implicitly only supports the "master" brand.
+
+When promoted by `obt init` in [part one](/docs/tutorials/create-a-new-component-part-1) we select all brands, so the `origami.json` file of our component should include an array of each brand `"brands": ["master","internal","whitelabel"],`. If not, update your `origami.json` now.
+
+### Switching Brands In Development
+
+When developing a branded Origami component pass the `--brand` flag to the `obt dev` command to switch brand.
+
+For example to build the `internal` version of our component run the command `obt dev --brand internal`. You should see in our demo the background colour has changed from a wheat colour to a light slate colour. That's because wheat is not part of the [internal brand colour palette](https://registry.origami.ft.com/components/o-colors@5.2.5/?brand=internal#demo-primary-palette). As we used a colour usecase `oColorsByUsecase('box', 'background')` in [part two](/docs/tutorials/create-a-new-component-part-2), rather than specify a specific colour, it was updated automatically for the internal brand.
+
+<figure>
+	<img alt="" src="/assets/images/tutorial-new-component/hello-world-demo-5-sass.png" />
+	<figcaption class="o-typography-caption">
+        Our example component for the internal brand no longer has a wheat background, because the background was set using a colour usecase.
+	</figcaption>
+</figure>
+
+Do the same for the whitelabel brand by running `obt dev --brand whitelabel`. You should see a Sass error `Error: 'The color "slate" does not exist.`. This error is because we set a border colour by name `oColorsByName('slate')` in [part two](/docs/tutorials/create-a-new-component-part-2), but slate is not in the limited [whitelabel colour palette](https://registry.origami.ft.com/components/o-colors@5.2.5/?brand=whitelabel#demo-primary-palette).
+
+To fix this error we need to set the border colour of our component differently depending on which brand is being used.
+
+### Configuring Brand Variables
+
+To style our component according to the current brand we need to include [o-brand](https://registry.origami.ft.com/components/o-brand@/readme) as a dependency. Install `o-brand` by running `bower install o-brand --save` and importing its Sass file in `main.scss`, as we did for other dependencies in [part two](/docs/tutorials/create-a-new-component-part-2).
```

Comment:
o-brand is already installed and included in the main.scss file due to telling `obt init` that we are making a branded component. Should we remove or reword this paragraph?